101-data-factory-v2-blob-to-postgresql-copy/azuredeploy.json

```diff
@@ -53,6 +73,8 @@
     }
   },
   "variables": {
+    "storageAccountName": "[last(split(parameters('storageAccountResourceId'), '/'))]",
+    "azurePostgreSqlDatabaseConnectionString": "[concat('Server=', parameters('postgreSqlServerName'),'.postgres.database.azure.com;Port=5432;Database=' ,parameters('postgreSqlDatabase'), ';UID=', parameters('postgreSqlUserName'), '@', parameters('postgreSqlServerName'), ';Password=', parameters('postgreSqlPassword'), ';SSL Mode=Require')]",
```

Comment:
use reference to get the FQDN - you'll need to move this from variables to where it's used below (reference cannot be used in a variable)

Reply:
sure. Used reference

Reply:
This still needs to be removed (and the one instance below updated...

Reply:
I have indeed removed it.